help/en/html/apps/StoringYourWork.shtml

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+
+<html lang="en">
+<head>
+
+  <title>JMRI: Storing Your Work</title>
+  <meta name="Author" content="Jerry Grochow">
+  <!--#include virtual="/Style.shtml" -->
+ 
+</head>
+<!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+  <div id="mBody">
+    <!--#include virtual="SidebarNew.shtml" -->
+    <div id="mainContent">
+
+        <a id="TopOfPage" name="TopOfPage"></a> 
+        <h1>JMRI: Storing Your Work</h1>
+        <p class="subtitle">What Everyone Should Know</p>
+        
+      <p class="noted">This page provides the basic informaiton you should know about what commands are used to store the
+      data you create and update in JMRI. A companion page with more information for advanced users about when, how, and where JMRI
+      stores data is available <a href="DataManagement.shtml"> here.</a></p>
+        
+      <p>As you work with JMRI, you will create and update many different items within its various applications and tools.
+      You may also develop and run "scripts" that will do that as well. JMRI may also update some of this data during train operaton.
+      To ensure that data you create (or is created and updated for you) is stored for your future use, you need to know when different
+      things get saved to a disk file (and, in the case of CVs, to your locomotives and other DCC decoders) so that they will be there
+      when you next enter JMRI.  There are also a few things that are saved automatically by JMRI, and some that are not saved at all
+      after you quit.</p>
+      
+       <ul class="snav">
+		<li><a href="#usave">User initiated</a></li>
+		<li><a href="#autosave">Automatic</a></li>
+		<li><a href="#nosave">Not stored</a></li>
+	   </ul>
+
+<!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+      <a name="usave" id="usave"></a>
+      <h2>Things you need to "store" or "save"</h2>
+ 
+      <div class="para">      
+      
+      <p>Unfortunately, there is no single command to store all user created or modified data, and JMRI does not always notify you if there
```

Comment:
"JMRI is meant to have a simple model for handing the information it needs to do its various functions. Certain kinds of data are handled automatically.  For other kinds, were you might want to decide whether or not to change the data that's kept from one run of JMRI to the next, we provide options that let you control that by function.  For example, saving changes to roster entries is separate from deciding to store away the entire layout configuration".  There's nothing unfortunate about this at all.

Reply:
If the goal is to help people understand how JMRI works, would it help to start by explaining why it works this way?  The existing text here is reinforcing word-processor thinking by starting with that as an explanation.

Reply:
Bob - I integrated your conceptual approach into the page preamble and also moved some other text around.  If you think this is sufficient for now, I suggest we now get this out for the developer community to comment further.  Note that this page is not yet linked anywhere so only those with the URL will find it.  I’ll put a post to the JMRI-developers group when it is online.  Thanks, Jerry